spec.html

```diff
@@ -20352,10 +20351,10 @@ <h1>Static Semantics: BoundNames</h1>
       </emu-alg>
       <emu-grammar>GeneratorDeclaration : `function` `*` `(` FormalParameters `)` `{` GeneratorBody `}`</emu-grammar>
       <emu-alg>
-        1. Return &laquo; *"\*default\*"* &raquo;.
+        1. Return &laquo; ~default~ &raquo;.
```

Comment:
I believe this still ends up making a binding named `~default~` in [InitializeEnvironment of Source Text Module Records](https://ci.tc39.es/preview/tc39/ecma262/sha/870e5f7cc27ac0e7489059d3c650595b100a2796/#sec-source-text-module-record-initialize-environment), step 23.